src/layman/celery.py

```diff
@@ -47,6 +47,11 @@ def task_postrun(workspace, publication_type, publication_name, task_id, task_na
             last_task_id = chain_info['last']
             finish_publication_chain(last_task_id)
             clear_steps_to_run_after_chain(workspace, publication_type, publication_name)
+    info = layman_util.get_publication_info(workspace, publication_type, publication_name, context={'keys': ['name']})
+    if not info:
+        current_app.logger.warning(f"POST task={task_name}, workspace={workspace}, publication_type={publication_type},"
+                                   f"publication_name={publication_name} Have to delete publication once more.")
```

Comment:
`Have to delete publication once more` => `Publication does not exist, so we delete it`